src/govuk/i18n.unit.test.mjs

```diff
@@ -2,55 +2,57 @@ import { I18n } from './i18n.mjs'
 
 describe('I18n', () => {
   describe('.t', () => {
-    let config = {}
+    /** @type {import('./i18n.mjs').TranslationsFlattened} */
+    let translations = {}
```

Comment:
Renamed this because the 2nd param is called `config` (the 1st is `translations`)

Was a bit confusing

```mjs
export function I18n (translations, config) {
```